### PR TITLE
Change all calls to messaging.new_message(s) to init in 1 line

### PR DESCRIPTION
**Pull request by sshane**
This PR complements commaai/cereal#35

### CODE REVIEW

File: selfdrive/locationd/locationd.py [hunk 1]
```diff
@@ -183,10 +183,9 @@ def locationd_thread(sm, pm, disabled_logs=[]):
 
     if localizer.filter_ready and sm.updated['gpsLocationExternal']:
       t = sm.logMonoTime['gpsLocationExternal']
-      msg = messaging.new_message()
+      msg = messaging.new_message('liveLocation')
       msg.logMonoTime = t
 
-      msg.init('liveLocation')
```
Comment (ShaneSmiskol) [inline selfdrive/locationd/locationd.py]: This is causing the merge conflict. Does it matter that you initialize the message after setting the `logMonoTime`?

Comment (pd0wm) [reply selfdrive/locationd/locationd.py]: No that doesn't matter.

**Comment by geohot** [comment]
Now we are talking. Love the 55 line loss!

**Comment by ShaneSmiskol** [comment]
I think I was able to resolve the merge conflict, let me know if it looks good!

**Comment by pd0wm** [comment]
Just bumped cereal on master. Can you rebase?

**Comment by ShaneSmiskol** [comment]
Think that should do it

**Review by pd0wm — APPROVED** [review]

**Pull request merged**